src/components/@molecules/TransactionDialogManager/stage/TransactionStageModal.tsx

```diff
@@ -476,7 +477,12 @@ export const TransactionStageModal = ({
           !!requestError ||
           isTransactionRequestCachedData
         }
-        onClick={() => sendTransaction(request!)}
+        onClick={() => {
+          sendTransaction(request!)
+          trackRegistrationEvent(
+            actionName === 'commitName' ? 'Commit Wallet Opened' : 'Finish Wallet Opened',
```

Comment:
You will need to add a condition before this or it will send a track event for every transaction.